.github/workflows/test.yml

```diff
@@ -22,4 +28,17 @@ jobs:
           key: v1/${{ runner.os }}/node-${{ matrix.node }}/${{ hashFiles('**/yarn.lock') }}
           restore-keys: v1/${{ runner.os }}/node-${{ matrix.node }}/
       - run: yarn
+      - name: Set up @percy/cli master
```

Comment:
```suggestion
      - name: Set up @percy/cli ${{ github.event.inputs.branch }}
```

try this, if this doesn't work remove `master` since this can be any branch